Source/WebInspectorUI/UserInterface/Views/DataGrid.css

```diff
@@ -63,11 +64,13 @@
 }
 
 .data-grid.inline {
-    border: 1px solid var(--border-color);
+    border: var(--data-grid-header-border-size) solid var(--border-color);
 }
 
 .data-grid > .header-wrapper {
-    border-bottom: 1px solid var(--border-color);
+    position: absolute;
```

Comment:
I'm not sure I follow why this has to now be absolutely positioned as well?

Reply:
That's to ensure the DataGrid header is placed on top of the absolutely-positioned DataGrid content.
Any oversize content will ensure the header renders over the content and not underneath it and looks clipped.

Reply:
@dcrousso Which lines are you referring to?
I can't identify a difference between this patch (left) and system Safari (right): ([direct link to image](https://github.com/WebKit/WebKit/assets/105809798/ed57dc95-fab1-4c4d-8593-b6b410ef48cc))

<img width="2560" alt="Screenshot 2024-05-03 at 10 55 50" src="https://github.com/WebKit/WebKit/assets/105809798/ed57dc95-fab1-4c4d-8593-b6b410ef48cc">

Reply:
you need to look at a view that has a `WI.TimelineRuler` (e.g. Network Requests, the overview when no timeline is selected, etc.)